spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -290,15 +290,15 @@
       let(:phone_confirmed) { false }
 
       it 'prompts to finish verifying profile, then redirects to SP' do
-        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return(true)
-
+        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return true
```

Comment:
I meant that both of the changes in this file are not related to this PR. They are only cosmetic changes. One removes the parentheses around `true`, and the other is removing a blank line.